pydarn/plotting/fan.py

```diff
@@ -107,6 +102,9 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, scan_index: int = 1,
                 the label that appears next to the colour bar.
                 Requires colorbar to be true
                 Default: ''
+            kawrgs: key = value
```

Comment:
I usually reserve kwargs for matplotlib functions as you can change linewidth and whatnot. Hem actually wouldn't be needed since we know the Station Id. See above comment. This means less user error.